iac-extensions/arm/src/main/java/org/sonar/iac/arm/parser/bicep/BicepLexicalConstant.java

```diff
@@ -43,6 +43,8 @@ public class BicepLexicalConstant {
   public static final String QUOTED_STRING_LITERAL_NO_QUOTES = "(?:(?!'|\\$\\{)[\\s\\S])*+";
   public static final String IDENTIFIER_LITERAL = "[a-zA-Z_][a-zA-Z_0-9]*+";
   public static final String AMBIENT_TYPE = "(?:array|bool|int|object|string)(?![\\S])";
+  public static final String UNARY_OPERATOR = "!|-|\\+";
+  public static final String MULTILINE_STRING = "(?:(?!''')(?:[\\s\\S])?+)*+";
```

Comment:
I'm always fascinated about this regex magic. Nice to exclude previous triple apostrophe.

Reply:
It was tricky